package.json

```diff
@@ -83,9 +81,9 @@
   },
   "scripts": {
     "build": "webpack --config webpack.config.production.js",
-    "prerender": "SCRIVITO_PRERENDER=true npm run build && node prerender/storePrerenderedContent.js",
+    "prerender": "SCRIVITO_PRERENDER=true npm run build && node buildNode/prerender_content.js",
```

Comment:
I did some benchmarking. Interestingly the node version is slower than the headless-chrome version. Node also seems to consume _more_ memory then headless-chrome - but this could be a measurement glitch.

How it I test this?

I did three runs with `SCRIVITO_ORIGIN=http://localhost:8080 /usr/bin/time -l npm run prerender` on `master` and on your branch.

Here are the results (all numbers are milliseconds except for "Max RSS", which is "maximum resident set size" reported by `time -l`:

|             | **Time (master)** | **Time (node)** | **Webpack (master)** | **Webpack (node)** | **Prerender-Script (master)** | **Prerender-Script (node)** | **Max RSS (master)** | **Max RSS (node)** |
| ----------- | ----------------- | --------------- | -------------------- | ------------------ | ----------------------------- | --------------------------- | -------------------- | ------------------ |
| **Run 1**   | 81.960            | 95.820          | 20.419               | 23.917             | 10.321                        | 10.489                      | 2.016.235.520        | 2.445.258.752      |
| **Run 2**   | 79.590            | 96.400          | 19.665               | 24.665             | 7.650                         | 9.409                       | 2.058.100.736        | 2.382.045.184      |
| **Run 3**   | 78.220            | 96.960          | 19.576               | 24.138             | 7.238                         | 9.252                       | 1.934.454.784        | 2.389.827.584      |
| **Average** | **79.923**        | **96.393**      | **19.887**           | **24.240**         | **8.403**                     | **9.717**                   | **2.002.930.347**    | **2.405.710.507**  |

Reply:
Interesting results. However, on my machine, with 3 runs each, I get different results for **Prerender-Script** (looking at `[storePrerenderedContent]: ?s` and `[prerenderContent]: ?s`):

Node: x̄ = 8.8993
Browser: x̄ = 9.0993